dashboard/src/components/users/UserSkillsPerformed.vue

```diff
@@ -35,6 +36,12 @@ limitations under the License.
             <b-button variant="outline-info" @click="applyFilters" data-cy="performedSkills-filterBtn"><i class="fa fa-filter"/> Filter</b-button>
             <b-button variant="outline-info" @click="reset" class="ml-1" data-cy="performedSkills-resetBtn"><i class="fa fa-times"/> Reset</b-button>
           </div>
+          <div class="col">
```

Comment:
Not a huge deal but I think instead of using custom style could apply bootstrap util classes to the parent div, maybe something like ``text-right mr-2``